### DISCUSSION TITLE
Java-Language-Server not showing warnings/errors?

### DISCUSSION THREAD

**Discussion by MATOO-Dev** [nix-community/nixvim#1487] [Q&A]
I'm trying to set up LSPs on NixOS as a home-manager module. Currently my config.plugins.lsp looks like this:
```nix
lsp = {
    enable = true;
    servers = {
        java-language-server.enable = true;
        nil_ls.enable = true;
    };
};
```
The Nix language server functions just fine, but the Java one doesn't show any warnings/errors. Is there an option I'm missing or do I need to install an additional package or something?

`:LspInfo` returns the following:
![Screenshot from 2024-05-05 13-17-38](https://github.com/nix-community/nixvim/assets/52349884/d95a992f-694f-4c6a-acdd-a9e76ce5f2ae)

**Comment by ZainKergaye** [comment]
I hope you got it fixed but I had the same issue. Look at my nix dotfiles. Specifically User/Programs/Nixvim/lsp.nix. That is what I have so far and now I have java-language-server and jdtls running at the same time. I still need to tweak this but two LSP's is better than 0. Still working on the DAP too lol
